config/samples/deployment.yaml

```diff
@@ -1,219 +0,0 @@
-apiVersion: v1
```

Comment:
It seems that one step has failed since the deployment is removed. Let's have a look at this on Monday.